lib/src/animate.dart

```diff
@@ -176,6 +177,9 @@ class Animate extends StatefulWidget with AnimateManager<Animate> {
   /// ```
   final double? target;
 
+  // TODO: add documentation
```

Comment:
Appreciate if you take a first stab at this.

Reply:
Will gladly do this tomorrow :)